test/chi/chi-vue/data-table-portal.pug

```diff
@@ -198,6 +198,11 @@ script.
         ],
       },
     },
+    mounted(){
+      if (window.Cypress) {
+        window.portalAccordionDataTable = this;
```

Comment:
Do you need this definition to access it later? I don't see how you use it

Reply:
yes i believe i use it in the spec to test the portal accordion event